src/main/resources/l10n/JabRef_ko.properties

```diff
@@ -1849,7 +1849,7 @@ Add\ new\ String=문자열 추가
 Must\ not\ be\ empty\!=비워둘 수 없습니다\!
 Open\ Help\ page=도움말 열기
 Add\ new\ field\ name=새 필드 이름 추가
-Field\ name\:=필드 이름\: 
+Field\ name\:=필드 이름\:
```

Comment:
Does this have something to do with your code changes?

Reply:
Hi, I have already deleted all the translations in the non-English file. Sorry for that, next time I definitely communicate with others first. I'm not sure what is the reason for labelling the Filename format and Field name column in red. I checked them in my code, and there is no whitespace or something.

Reply:
Hi, I have already deleted all the translations in the non-English file. I'm not sure what is the reason for labelling the Filename format and Field name column in red. I checked them in my code, and there is no whitespace or something.